main.css

```diff
@@ -78,3 +78,109 @@ p {
 li {
   margin-right: 15px;
 }
+
+.second-main {
+  display: grid;
+  align-items: center;
+  background: #e5e5e5;
+}
+
+.line {
+  margin-bottom: 80px;
+}
+
+h4 {
+  font-size: 42px;
+  font-weight: lighter;
+}
+
+h5 {
+  font-size: 32px;
+}
+
+.par-black {
+  color: #1c1a19;
+  margin: 0;
+}
+
+.activ-g {
+  background-image: url("pictures/ImgPlaceholder2.png");
+  background-repeat: no-repeat;
+  margin-top: 20px;
+  margin-bottom: 60px;
+}
+
+h6 {
+  font-size: 32px;
+  margin-bottom: 0%;
+  margin-left: 20px;
```

Comment:
- [ ] Please keep your code DRY by creating a class for redundant CSS attributes. In this case, `margin-left: 20px` appears 6 times. You may use this tool to check for CSS redundancy in the future: https://courses.cs.washington.edu/courses/cse154/20su/resources/assets/css-redundancy-checker/index.html
![image](https://user-images.githubusercontent.com/2739245/125143344-09fffe80-e0e0-11eb-8a5c-7e68ca6923b1.png)